x-pack/plugins/observability_solution/synthetics/server/routes/default_alerts/default_alert_service.ts

```diff
@@ -180,7 +192,7 @@ export class DefaultAlertService {
           actions,
           name: alert.name,
           tags: alert.tags,
-          schedule: alert.schedule,
+          schedule: { interval },
```

Comment:
With this change, every time we update an default alert, we'll use the minimum schedule. That means that if a user has changed the interval at all, we'll change it back on rule updates. Auto updates in particular happens when making any changes on the setting page. Is this desired or should we add logic checking if the existing interval is less than the minimum range and only set it to the minimum if true?

Reply:
Adjusted the logic to use the existing interval if it's higher than the default.